### ISSUE TITLE
Spike: Upgrade all generated manifests to IIIF v3 manifest spec

### ISSUE THREAD

**Issue by jeremyf** [scientist-softserv/britishlibrary#225]
IIIF version 3’s implementation of UV does not natively support thumbnails anymore (as we found out a few months back).  The solution would be to upgrade the IIIF Manifest gem to 1.2.0 which has this feature.

From @kirkkwang: "Looking ahead, audio/video thumbnails do not work as of 1.2.0, that’s something I need to fix when the time comes with UTK still."

Depends on #214 

upgrade gem?

**Comment by ShanaLMoore** [comment]
Heads UP: there's a hard dependency in hyrax. 

OPTIONS (per jeremy)
- make a fork of iiif_manifest and manually change the version number on the repo of where it's at now and point it to that repo change. 
- otherwise, upgrade hyrax to 3+ which may be out of scope
- we may be able to upgrade them to hyrax 2.9.6

**Comment by jillpe** [comment]
Goal is to figure out how to upgrade smoothly
